### PR TITLE
support domain (security) point of contact fields

### PR DESCRIPTION
**Pull request by jimmytsang**
This pr introduces the new configurable domain meta fields "Point of Contact" and "Security Point of Contact" 

- user can add a (security) point of contact in any page that shows domain details 
- successfully passes new unit and functional tests to safeguard this new feature

Key files to review
- DomainDetails.js
- AddPoc.js
- domain.spec.js

### CODE REVIEW

File: ui/src/components/member/AddPoc.js [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+import styled from '@emotion/styled';
+import InputLabel from '../denali/InputLabel';
+import InputDropdown from '../denali/InputDropdown';
+import { colors } from '../denali/styles';
+import { selectAllUsers } from '../../redux/selectors/user';
+import { USER_DOMAIN } from '../constants/constants';
+import RequestUtils from '../utils/RequestUtils';
+
+const SectionsDiv = styled.div`
+    width: 760px;
```
Comment (jchen05-yh) [inline ui/src/components/member/AddPoc.js:29]: Would `max-width` make it responsive?

File: ui/src/components/member/AddPoc.js [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+        let meta = {
+            contacts: this.props.contacts,
+        };
+        meta.contacts[this.props.contactType] = this.state.pocName;
+        let domainName = this.props.domain;
+        let auditMsg =
+            'Updating Point of Contact ' +
+            this.state.pocName +
+            ' for domain ' +
+            this.props.domain;
```
Comment (jchen05-yh) [inline ui/src/components/member/AddPoc.js:86]: Maybe consider using `Updating Point of Contact ${this.state.pocName} for domain ${this.props.domain}`?

File: ui/src/components/member/AddPoc.js (hunk 1 above)
Comment (jimmytsang) [reply ui/src/components/member/AddPoc.js:29]: the styling properties are copied from add member modal, just tested the AddPoc modal is responsive to window resize.

File: ui/src/components/member/AddPoc.js (hunk 2 above)
Comment (jimmytsang) [reply ui/src/components/member/AddPoc.js]: good suggestion, fixed

**Review by dantewuuuuu — APPROVED** [review]

**Pull request merged**